docs/next/guides/building-and-testing/modules.md

```diff
@@ -481,352 +275,822 @@ new Handsontable(container, {});
+```
+
+#### Step 3: Import your editor module
+
+Import an editor module of your choice. For example:
+
+```js
+import Handsontable from 'handsontable/base';
+import 'handsontable/dist/handsontable.full.css';
+
+// import the `PasswordEditor` module
 import { registerEditor, PasswordEditor } from 'handsontable/editors';
 ```
 
-Afterwards you need to use the `registerEditor` method to register `PasswordEditor`:
+#### Step 4: Register your editor
+
+Register your editor, using the `registerEditor()` method. For example:
 
 ```js
-// register the editor before using it
+import Handsontable from 'handsontable/base';
+import 'handsontable/dist/handsontable.full.css';
+import { registerEditor, PasswordEditor } from 'handsontable/editors';
+
+// register the `password` editor
 registerEditor(PasswordEditor);
 ```
 
-Now, you can use `PasswordEditor`, the full example looks like this:
+::: tip
+#### Registering all editors at once
+
+If you import all of Handsonable's editor modules, register them all with the `registerAllEditors()` method:
+
+```js
+// import the `registerAllEditors()` method
+import { registerAllEditors } from 'handsontable/registry';
+
+// register all editors
+registerAllEditors();
+```
+:::
+
+#### Step 5: Use your editor
+
+Now, you can start using your editor, by setting the [`editor`](@/api/options.md#editor) [configuration option](@/guides/getting-started/setting-options.md):
 
 ```js
 import Handsontable from 'handsontable/base';
+import 'handsontable/dist/handsontable.full.css';
 import { registerEditor, PasswordEditor } from 'handsontable/editors';
 
 registerEditor(PasswordEditor);
 
-// use the editor as needed
+// an example
 new Handsontable(container, {
   columns: [
-    {
+    { 
+      // use the `password` editor
       editor: 'password',
     },
   ]
-  // rest of the settings
 });
 ```
 
-And that is all! You can use the password editor!
+### Using renderer modules
+
+Each renderer module contains a different type of [cell renderer](@/guides/cell-functions/cell-renderer.md):
+
+| Module                 | Cell renderer alias |
+| ---------------------- | ------------------- |
+| `autocompleteRenderer` | `autocomplete`      |
+| `baseRenderer`         | `base`              |
+| `checkboxRenderer`     | `checkbox`          |
+| `htmlRenderer`         | `html`              |
+| `numericRenderer`      | `numeric`           |
+| `passwordRenderer`     | `password`          |
+| `textRenderer`         | `text`              |
 
-#### Importing all editors at once
+To start using a renderer module, follow the steps below.
 
-You can quickly import and register all editors at once:
+#### Step 1: Import the required modules
+
+First, make sure to [import the required modules](#importing-required-modules):
 
 ```js
-// import the `registerAllEditors()` function
-import { registerAllEditors } from 'handsontable/registry';
+// import the `handsontable/base` module
+import Handsontable from 'handsontable/base';
```

Comment:
I don't see how a comment like the one on line 543 is useful. I think that obvious things should not be stated in code comments.

Reply:
1) It points at the line that we're currently describing or focusing on.
2) It matches the pattern of other code examples (usually longer ones, though) in the docs, where comments like this point you to the currently-described line ([for example](https://handsontable.com/docs/column-summary/#step-1-enable-the-columnsummary-plugin) – some docs do this by tooltips as well, [for example](https://stripe.com/docs/connect/collect-then-transfer-guide?platform=web#create-an-express-account-and-prefill-information)).
3) Generally: what may seem obvious to us, may not be obvious to the recipient reaching for a detailed tutorial.

But it's your call, I'm removing them: bc4bed1

Reply:
Thanks. I agree that it makes sense to draw attention to a single line in a long snippet. But in a short snippet, it just repeats the same information that can be found in the snippet description.